README.md

```diff
@@ -1,14 +1,26 @@
-> As of version `3.0.0` this library exposes [ES modules](http://exploringjs.com/es6/ch_modules.html#sec_basics-of-es6-modules). Use an ES module aware bundler such as [Webpack](https://webpack.js.org) or [Rollup](https://rollupjs.org) to bundle it for the browser.
-
-
 # Camunda Modeler Plugin Helpers :electric_plug:
 
 [![Compatible with Camunda Modeler version 2.2](https://img.shields.io/badge/Camunda%20Modeler-2.2+-blue.svg)](https://github.com/camunda/camunda-modeler)
 
-
 Helper functions for bundling your Camunda Modeler client plugins.
 
-## How to use
+
+## Usage
+
+Install the plug-in:
+
+```
+npm install camunda-modeler-plugin-helpers
+```
+
+Use any of the [exposed utilities](#api).
+
+> [!WARNING]
+> This library re-exposes core utilities that you want to use as you develop plug-ins for the [Camunda Modeler](https://github.com/camunda/camunda-modeler).
+> If you use, i.e. React within your plug-in, you _must_ consume it via the [plug-in export](#react). Use our [webpack plug-in](https://github.com/camunda/camunda-modeler-webpack-plugin) to accomplish that automatically, or use a source mod / configure your bundler accordingly as you build plug-ins.
```

Comment:
```suggestion
> If you use React within your plug-in, you _must_ consume it via the [plug-in export](#react). Use our [webpack plug-in](https://github.com/camunda/camunda-modeler-webpack-plugin) to accomplish that automatically, or use a source mod / configure your bundler accordingly as you build plug-ins.
```

Reply:
@barmac This message is too tightly attached to `react`, I used it as an example, on purpose. Many other components must also be re-written (properties panel, preact, ...), and the message should indicate that (1) this has to happen and (2) the webpack plug-in is the on stop shop to do it.

Reply:
Incorporated via https://github.com/camunda/camunda-modeler-plugin-helpers/pull/30/commits/338c1ca4642503ac62879eeaea4eb0849748193d, please have another look.